lib/workers/pr/body/config-description.ts

```diff
@@ -9,7 +9,7 @@ export async function getPrConfigDescription(
   prBody += emojify(`:date: **Schedule**: `);
   if (
     config.schedule &&
-    config.schedule !== 'at any time' &&
+    (config.schedule as never) !== 'at any time' &&
```

Comment:
What's the benefit of `as never` here? I'm not super familiar with `never`.

Reply:
It is equivalent to `any`. `schedule` is migrated to `string[]`